client/components/data/query-activity-log/README.md

```diff
@@ -1,24 +1,47 @@
-Query Activity Log
-==================
-
-A query component for fetching activity log data.
-
-## Usage
-
-```
-<QueryActivityLog siteId={ siteId } />
-```
-
-This component renders nothing. It ensures that any activity logs for the given site are available in global state.
-
-
-## Props
-
-### `siteId`
-
-<table>
-	<tr><th>Type</th><td>Number</td></tr>
-	<tr><th>Required</th><td>Yes</td></tr>
-</table>
-
-Site to fetch activities for.
+Query Activity Log
+==================
+
+A query component for fetching activity log data.
+
+## Usage
+
+```
+<QueryActivityLog siteId={ siteId } />
+```
+
+This component renders nothing. It ensures that any activity logs for the given site are available in global state.
+
+
+## Props
+
+### `siteId`
+
+Type   | Required
+------ | --------
+Number | Yes
+
+Site to fetch activities for.
+
+### `dateEnd`
+
+Type   | Required
+------ | --------
+Number | No
+
+Unix millisecond timestamp to fetch activity on or before.
```

Comment:
I prefer to leave this as is. It's not too hard to imaging that omitting limits gives unlimited results, and I want to avoid describing in too much detail how the API will behave.